src/main/java/com/gmail/inverseconduit/utils/ChatBuilder.java

```diff
@@ -0,0 +1,196 @@
+package com.gmail.inverseconduit.utils;
+
+import com.gmail.inverseconduit.datatype.ChatMessage;
+
+/**
+ * Helper class for building chat messages that have SO Chat markdown.
+ * @author Michael Angstadt
+ * @see <a href="http://chat.stackoverflow.com/faq#formatting">Formatting
+ * FAQ</a>
+ */
+public class ChatBuilder implements CharSequence {
```

Comment:
Eh, it should be the programmer's responsibility to make sure the syntax is correct.  Adding validation would unnecessarily complicate the class, IMO.